src/NuGet.Clients/VisualStudioAPI/IPackageManagerProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Interface for integrating other package manager.eg:bower,npm
+    /// Contains methods to check packages in intergated package manager and direct to intergated package manager. 
+    /// </summary>
+    [ComImport]
+    [Guid("BCED5BF2-40FC-4D9F-BF0A-43CD4E9FF65F")]
+    public interface IPackageManagerProvider
+    {
+        /// <summary>
+        /// Package manager name.
+        /// </summary>
+        string PackageManagerName { get; }
+
+        /// <summary>
+        /// Package manager unique id.
+        /// </summary>
+        string PackageManagerID { get; }
+
+        /// <summary>
+        /// Description for package manager.
```

Comment:
Again, as a consumer I don't know what to do with this, where does it gets displayed what it is for.